migrations/20220523141538_recipe_transactions_table.js

```diff
@@ -1,26 +1,21 @@
 exports.up = async (knex) => {
-  // check extension is not installed
-  const [extInstalled] = await knex('pg_extension').select('*').where({ extname: 'uuid-ossp' })
-
-  if (!extInstalled) {
-    await knex.raw('CREATE EXTENSION "uuid-ossp"')
-  }
-
   const uuidGenerateV4 = () => knex.raw('uuid_generate_v4()')
   const now = () => knex.fn.now()
 
   await knex.schema.createTable('recipe_transactions', (def) => {
     def.uuid('id').defaultTo(uuidGenerateV4())
     def.integer('token_id').unsigned().notNullable()
     def.uuid('recipe_id').notNullable()
-    def.enu('status', ['submitted', 'minted'])
+    def.enu('type', ['Creation']).notNullable()
+    def.enu('status', ['Created', 'Submitted', 'Rejected', 'Amended', 'Accepted'])
```

Comment:
this and `status` in open-api spec for the response don't match https://github.com/digicatapult/inteli-api/blob/51c95d399a5105aa9eba5eee2b11a478347b1cae/app/api-v1/api-doc.js#L317